### ISSUE TITLE
Bury googlemaps API key in config.js

### ISSUE THREAD

**Issue by av2335** [glenpierce/UBHub#33]
The API key for googlemaps in maps.jade and a few other files looks real.  This makes it available to anyone who happens to have access to our github.

Not sure who that might be, but it's a good practice to hide that, just in case.  The usual way is to store APIs in config.js (which is gitignored) and add those keys using the variable name in any public file.

Unless anyone sees a reason to not do this, I could go ahead and code it up.  Should be simple.

**Comment by glenpierce** [comment]
Yes, please do that. That API key isn't a sensitive one since it will be visible to users on the page itself and I've locked down the API key to only be usable on the site's IP address, but this would be preferable.
